tests/functional/kmip/serverside_encryption.js

```diff
@@ -68,37 +75,185 @@ function _buildS3() {
 }
 const s3 = _buildS3();
 
+function _putObject(bucketName, objectName, encrypt, cb) {
+    const params = {
+        Bucket: bucketName,
+        Key: objectName,
+        Body: 'I am the best content ever',
+    };
+
+    if (encrypt) {
+        Object.assign(params, {
+            ServerSideEncryption: 'AES256',
+        });
+    }
+
+    s3.putObject(params, err => {
```

Comment:
super minor, but you could just pass `cb` as the final argument instead of the `err => ...` function